.github/workflows/test.yml

```diff
@@ -41,9 +41,6 @@ jobs:
           - name: "static batching"
             markers: "cpu and decoder and not cb"
             flags: "--timeout=300"
-          - name: "quantized"
-            markers: "cpu and quantized"
-            flags: "--timeout=300"
```

Comment:
maybe we could leave the `quantized` flag to only run tests with the quantized model?

Reply:
That could also potentially speed things up as we can run the test with quantized model in parallel